packages/client/src/views/SysAdmin/Config/Certificates/previewDummyData.ts

```diff
@@ -367,14 +367,16 @@ const dummyBirthRegistrationResponse = {
   birthType: 'SINGLE',
   questionnaire: null,
   eventLocation: {
-    type: 'HEALTH_FACILITY',
+    id: 'f0517084-664e-434c-8ed3-2dc58ceabcfb',
+    type: 'PRIMARY_ADDRESS',
     address: {
-      line: [],
-      district: null,
-      state: null,
+      type: 'PRIMARY_ADDRESS',
+      line: ['', '', '', '', '', '', 'URBAN'],
+      district: 'ecc5a78b-e7d9-4640-ac65-e591a6a9590f',
+      state: 'df669feb-61a3-4984-ab24-4b28511b472a',
```

Comment:
As above.  Replace these IDs with a function that returns district and state IDs from the logged in userDetails own district and state